src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Subscription/NewAzureServiceBusSubscription.cs

```diff
@@ -152,15 +144,7 @@ public override void ExecuteCmdlet()
 
             if (RequiresSession != null)
                 subAttributes.RequiresSession = RequiresSession;
-
-#pragma warning disable 612, 618
-            if (IsReadOnly != null)
-                subAttributes.IsReadOnly = IsReadOnly;
-            if (DeadLetteringOnFilterEvaluationExceptions != null)
-                subAttributes.DeadLetteringOnFilterEvaluationExceptions = DeadLetteringOnFilterEvaluationExceptions;
```

Comment:
Why are you removing the IsReadOnly parameter but not DeadLetteringOnFilterEvaluationExceptions parameter?

Reply:
removed the parameters